docs/source/docker/configuration/configuration.rst

```diff
@@ -180,20 +176,7 @@ The following are parameters which can be passed to the container:
   # optional deterministic unique output subdirectory for run, in place of timestamp
   run_directory:
 
-Additionally, you can pass all arguments which can be passed to the lightly CLI tool with the `lightly` prefix.
-For example,
-
-.. code-block:: console
-
-    docker run --rm -it \
-        -v {INPUT_DIR}:/home/input_dir:ro \
-        -v {OUTPUT_DIR}:/home/output_dir \
-        lightly/worker:latest \
-        token=MYAWESOMETOKEN \
-        lightly.loader.batch_size=512
-
-sets the batch size during training and embedding to 512. You find a list of all
-lightly CLI parameters here: :ref:`ref-cli-config-default`
+To get an overview of all available lightly config parameters check out the :ref:`ref-cli-config-default`
```

Comment:
of all available lightly config parameters -> of all possible configuration parameters of Lightly, please check out